evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -458,36 +459,46 @@ def test_component_graph_fit_features(
     mock_X_t = pd.DataFrame(np.ones(pd.DataFrame(X).shape))
     mock_fit_transform.return_value = mock_X_t
     mock_fit.return_value = Estimator
-    mock_predict.return_value = pd.Series(y)
+    mock_predict_proba.return_value = pd.DataFrame(y)
+    mock_predict_proba.return_value.ww.init()
 
     component_graph.fit_features(X, y)
 
     assert mock_fit_transform.call_count == 3
     assert mock_fit.call_count == 2
-    assert mock_predict.call_count == 2
+    assert mock_predict_proba.call_count == 2
 
 
 @patch("evalml.pipelines.components.Estimator.fit")
+@patch("evalml.pipelines.components.Estimator.predict_proba")
 @patch("evalml.pipelines.components.Estimator.predict")
-def test_predict(mock_predict, mock_fit, example_graph, X_y_binary):
+def test_predict(mock_predict, mock_predict_proba, mock_fit, example_graph, X_y_binary):
     X, y = X_y_binary
+    mock_predict_proba.return_value = pd.DataFrame(y)
```

Comment:
Bahaha RE comment to change output to dataframes, I'm curious about when it's a df and has multiple columns! but not a big change :)

Reply:
Oops I see 😅. I added the two test cases with different `predict_proba` outputs to address this.